projects/detr3d/detr3d/transform_3d.py

```diff
@@ -0,0 +1,12 @@
+from mmdet3d.registry import TRANSFORMS
+
+
+@TRANSFORMS.register_module()
+class filename2img_path:
```

Comment:
Seems that this transform is not used, can we delete this file?

Reply:
I deleted, but this might be a bug. In Transform: class Pack3DDetInputs we have 'img_path' in meta_keys. However, there is no such key when Pack3DDetInputs is processing the data. Instead, there is 'filename' which might has same meaning to 'img_path'